src/components/common/ColonyHome/hooks.ts

```diff
@@ -317,3 +311,42 @@ export const useDashboardHeader = (): ColonyDashboardHeaderProps => {
     setLeaveColonyConfirm,
   };
 };
+
+const calculateTotalFunds = async (
+  balances: ColonyBalances,
+  currency: SupportedCurrencies,
+) => {
+  const funds = balances.items
+    ?.filter(notNull)
+    .filter(({ domain }) => !!domain?.isRoot)
+    .reduce(async (total, { balance, token: { tokenAddress } }) => {
+      const currentPrice = await fetchCurrentPrice({
+        contractAddress: tokenAddress,
+        conversionDenomination: currency,
+      });
+
+      return (await total).add(BigNumber.from(balance).mul(currentPrice));
```

Comment:
Could this `currentPrice` be a decimal? Right now it would crash if that's the case

Reply:
Yep good idea